tests/toranj/cli/test-020-net-diag-vendor-info.py

```diff
@@ -69,17 +69,20 @@
 VENDOR_SW_VERSION_TLV = 27
 THREAD_STACK_VERSION_TLV = 28
 MLE_COUNTERS_TLV = 34
+VENDOR_APP_URL = 35
 
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 # Check setting vendor name, model, ans sw version
 
 r1.set_vendor_name('nest')
 r1.set_vendor_model('marble')
 r1.set_vendor_sw_version('ot-1.3.1')
+r1.set_vendor_app_url('example.com')
```

Comment:
This should be a proper URL including scheme, I think. E.g. `https://example.com/vendor-app`